src/main/webapp/app/overview/exercise-details/course-exercise-details.component.html

```diff
@@ -9,6 +9,8 @@
         <jhi-programming-exercise-student-ide-actions jhiIdeFilter [showInIDE]="true" [courseId]="courseId" [exercise]="exercise"> </jhi-programming-exercise-student-ide-actions>
         <jhi-exercise-details-student-actions jhiIdeFilter [showInIDE]="false" [courseId]="courseId" [exercise]="exercise" [showResult]="showResults && currentResult">
         </jhi-exercise-details-student-actions>
+        <jhi-updating-result class="mr-2" [result]="currentResult" [participation]="currentResult?.participation" [exercise]="exercise" [showGradedBadge]="true">
```

Comment:
This is not documented, but the @Input `result` doesn't do anything.
The `updating-result` expects the participation to have at least one result embedded and will use the result with the highest completionDate and assign it to `this.result`.

I know it's not part of this PR, but could we remove the Input annotation from result in the updating-result.component and remove the usages of this input from this component (I didn't see any other)? Please verify that it is working as expected.

Reply:
Removed the input annotation